great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -159,6 +221,14 @@ def validate_configuration(
             assert isinstance(
                 configuration.kwargs["quantile_ranges"], dict
             ), "quantile_ranges should be a dictionary"
+
+            if isinstance(
+                configuration.kwargs["quantile_ranges"]["value_ranges"], np.ndarray
+            ):
+                configuration.kwargs["quantile_ranges"][
+                    "value_ranges"
+                ] = configuration.kwargs["quantile_ranges"]["value_ranges"].tolist()
```

Comment:
This surprises me a bit to be here instead of managed by the configuration itself during serialization. Seems like the longer we can keep things in numpy the better.

Reply:
@jcampbell Unfortunately, the `all()` operator insists on lists and does not work on `numpy.ndarray` -- the conversion to list was in response to this error showing up when `numpy.ndarray` valued results were encountered by this configuration change.

Reply:
Got it. Totally a nitpick, but you could instead just check the resulting list for the check instead of overwriting the actual parameter.

Reply:
@jcampbell Nice catch -- fixed (and committed).  Thanks!

Reply:
@jcampbell Actually, I had to revert -- by leaving as `ndarray` I was getting serialization issues in the notebook.  Thanks.

Reply:
@jcampbell In the end, I was actually able to remove this conversion and get this validation to work the way it was written originally.  Thanks!